client/jetpack-cloud/sections/partner-portal/license-lightbox/license-lightbox-purchase-via-jetpackcom.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useContext, useCallback } from 'react';
+import SitesOverviewContext from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/context';
+import { getPurchaseURLCallback } from 'calypso/my-sites/plans/jetpack-plans/get-purchase-url-callback';
+import slugToSelectorProduct from 'calypso/my-sites/plans/jetpack-plans/slug-to-selector-product';
+import { SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+
+import './style.scss';
+
+type Props = {
+	productSlug: string;
+	onClose: () => void;
+	siteId?: number;
+	currentProduct: APIProductFamilyProduct;
+};
+
+const LicenseLightboxPurchaseViaJetpackcom: FunctionComponent< Props > = ( {
+	productSlug,
+	onClose,
+	siteId,
+	currentProduct,
```

Comment:
Technically we don't need to use (and perhaps shouldn't use) `currentProduct`, as there are two different products here, and we're buying the WP.com product (`productSlug`), not the JP Manage product (`currentProduct.slug`).

Reply:
👍 Thank you